src/components/BulkActions/BulkActions.tsx

```diff
@@ -17,7 +22,7 @@ import {CheckableButton} from '../CheckableButton';
 import {BulkActionButton} from './components';
 import styles from './BulkActions.scss';
 
-type BulkAction = DisableableAction;
+type BulkAction = DisableableAction & BadgeAction;
```

Comment:
The test uses an action that is a BadgeAction. Looks like it was added deliberatly so make sure the actual types match.